WordPress/src/main/res/values/strings.xml

```diff
@@ -140,14 +140,20 @@
     <string name="access_media_permission_required">Permissions required in order to access media</string>
     <string name="add_media_permission_required">Permissions required in order to add media</string>
     <string name="media_fetching">Fetching media…</string>
-    <string name="upload_queued">Queued</string>
     <string name="media_file_type">File type: %s</string>
     <string name="media_file_name">File name: %s</string>
     <string name="media_uploaded_on">Uploaded on: %s</string>
     <string name="media_dimensions">Dimensions: %s</string>
     <string name="media_upload_error">Media upload error occurred</string>
     <string name="media_generic_error">Media error occurred</string>
 
+    <string name="media_upload_state_queued">QUEUED</string>
```

Comment:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/5531/commits/fdc8139f48f9a400789d10ef3e4ffa49502cbe8e

@nbradbury Ignore my previous comments, I got confused about it all. It will work correctly when it's translated. Sorry about that!

Just to show what I am talking about:

![screenshot_1490704329](https://cloud.githubusercontent.com/assets/662023/24405006/f4c94872-13cb-11e7-82e7-3d92cba38cc4.png)
![screenshot_1490704236](https://cloud.githubusercontent.com/assets/662023/24405005/f4c72074-13cb-11e7-9190-2f0335e45135.png)